docs/source/considerations.rst

```diff
@@ -133,8 +133,8 @@ however you may notice some differences:
    instead. In this case, this class is named ``Greeter``.
 #. The ``Greeter`` class implements two methods:
 
-   #. ``on_init()``: which is executed on initialization and will, in this
-      case, simply bind a ``'PUSH'`` communication channel.
+   #. ``on_init()``: which is executed before the main loop starts, and will,
+      in this case, simply bind a ``'PUSH'`` communication channel.
```

Comment:
You need to revert the changes in the whole paragraph. I think you just changed back to "on_init", but forgot the description. :wink:

Reply:
Got me there 😀 .